client/app/core/config.js

```diff
@@ -1,4 +1,6 @@
 /* eslint angular/window-service: "off" */
+import { RootReducer } from '../reducers'
+import { autoRehydrate } from 'redux-persist'
```

Comment:
v5 of redux-persist seems to be moving away from autoRehydrate (https://github.com/rt2zz/redux-persist#migration-from-v4-to-v5)